index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>hello microverse</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1>Hello Microverse</h1>
```

Comment:
- [ ] I would kindly suggest you give your `h1` tag a class name and the name given to it should be used to style the content in your `CSS` file as required in the project requirement. 👍 👍